elcid/models.py

```diff
@@ -319,13 +334,6 @@ class Allergies(omodels.Allergies, ExternallySourcedModel):
     allergy_start_datetime = models.DateTimeField(null=True, blank=True)
     no_allergies = models.BooleanField(default=False)
 
-    @classmethod
-    def get_fieldnames_to_serialize(cls):
-        # Looks like we won't get these from the prescribing system
```

Comment:
Is this actually how we want allergies to serialize? 
Wasn't this useful - e.g. to not serialize the extra fields